agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/telemetry/TelemetryChannel.java

```diff
@@ -282,6 +300,11 @@ private CompletableResultCode internalSend(
     return result;
   }
 
+  // not including 401/403/503 in this list because those are commonly returned by proxy servers
+  // when they are not configured to allow traffic for westus-0
+  private static final Set<Integer> RESPONSE_CODES_INDICATING_REACHED_BREEZE =
+      new HashSet<>(asList(200, 206, 402, 408, 429, 439, 500));
```

Comment:
307/308

Reply:
redirect response codes only get here if they are of the infinite redirect loop variety

do infinite redirects count as "probably from breeze"? I'm good either way, can you update the spec to call that out though?

Reply:
![image](https://user-images.githubusercontent.com/56097766/163075743-69b7d2bb-a74e-41ed-a7f7-6763a51ac57e.png)

i did cover redirect in the spec here.

Reply:
ya, I'm asking specifically about "infinite redirect loops" (the ones that we cap at 10 redirects and then fail)

Reply:
that is handled by default. I mentioned that in the statsbeat spec.. not in "how to handle statsbeat failures" section though.

Reply:
you still haven't answered my question though:

> do infinite redirects count as "probably from breeze"?

Reply:
![image](https://user-images.githubusercontent.com/56097766/163076712-84f9490e-956e-47e4-a229-0111c47e8261.png)

Reply:
ok great, so we shouldn't add 307/308 here